internal/components/components.go

```diff
@@ -91,8 +91,8 @@ func FindMatchingStrings(components []string, r *regexp.Regexp) []string {
 }
 
 // GetSubcomponentPath creates a gNMI path based on the componnent name.
-func GetSubcomponentPath(name string) *tpb.Path {
-	if *deviations.GNOISubcomponentPath {
+func GetSubcomponentPath(name string, dut *ondatra.DUTDevice) *tpb.Path {
```

Comment:
Could you change the second argument to a boolean named "useNameOnly" and add a line to the godoc for this function that says something like "If useNameOnly is true, returns a path to the specified name instead of a full subcomponent path."

Reply:
done. And moved the deviation. GNOISubcomponentPath call in the test file.